### ISSUE TITLE
BUG: Post-Activation takes user to 'Sign Up' again, instead of "Sign In"

### ISSUE THREAD

**Issue by wanderslth** [flexiodata/flexio-web-app#581]
TL;DR -- after sign up and clicking the 'activate my account' button, you are activated but then returned to SIGN UP, rather than SIGN IN, which feels buggy and insecure:

1. Sign Up as normal:

![image](https://user-images.githubusercontent.com/25911712/70097318-b584cb00-15ee-11ea-82d8-45613ffab9f3.png)

2. Requires verification:

![image](https://user-images.githubusercontent.com/25911712/70097371-da793e00-15ee-11ea-9c2f-fb751ecc44ae.png)

3. Get the email:

![image](https://user-images.githubusercontent.com/25911712/70097389-e5cc6980-15ee-11ea-92be-f6b6d487bc1a.png)

4. Click the activation... spinner... **TAKES US BACK TO AN EMPTY-ISH SIGN UP FORM:**

![image](https://user-images.githubusercontent.com/25911712/70097411-f4b31c00-15ee-11ea-85f4-f051a262a5c8.png)

5. If I know to click "Sign In" we're all good:

![image](https://user-images.githubusercontent.com/25911712/70097435-0399ce80-15ef-11ea-8864-ed5824060698.png)

6. Enter password and I'm in:

![image](https://user-images.githubusercontent.com/25911712/70097457-114f5400-15ef-11ea-859c-01b52ee233d0.png)
